modules/amqp091/scanner.go

```diff
@@ -165,39 +173,27 @@ func (scanner *Scanner) Protocol() string {
 	return "amqp091"
 }
 
-func (scanner *Scanner) Scan(target zgrab2.ScanTarget) (zgrab2.ScanStatus, any, error) {
-	conn, err := target.Open(&scanner.config.BaseFlags)
+func (scanner *Scanner) GetDefaultDialerGroup() *zgrab2.DialerGroup {
```

Comment:
I think this should be more about getting default options from the module not getting a DailerGroup. Semantically, it doesn't make sense to ask a scanner for a Dialer.